packages/gossiplog/src/BranchIndex.ts

```diff
@@ -1,10 +1,10 @@
-import type { AbstractModelDB, ModelsInit } from "@canvas-js/modeldb"
+import type { AbstractModelDB, ModelSchema } from "@canvas-js/modeldb"
 
 export class BranchIndex {
 	public static schema = {
 		// $maxBranch is a singleton
 		$maxBranch: { id: "primary", maxBranch: "integer" },
```

Comment:
I'm surprised that a singleton maxBranch is sufficient! I expected it to need `$branches: { id: "primary", branch: "integer" }`.

But if `$messages` has an indexed `branch` property can't we just get the max branch at any time with `db.query("$messages", { select: { id: true, branch: true }, limit: 1, orderBy: { branch: "desc" } })`?

Reply:
Yeah you are right, we could just get this value by querying `$messages`. I'll update this.